shinysdr/i/network/webapp.py

```diff
@@ -50,13 +51,43 @@
 from shinysdr.values import SubscriptionContext
 
 
+# txWS is unfortunately dead upstream, and has buggy Python 3 support.
```

Comment:
:(

I have a dead branch that tries to move to Autobahn, but I was having trouble getting it to even show me any error stacks from my callbacks not working.